examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
Thanks for your questions, the more examples we have, and the more we implement the clearer it will get.